apollo-router/src/plugins/telemetry/metrics/apollo.rs

```diff
@@ -268,7 +77,9 @@ mod test {
             apollo_graph_ref: None,
             client_name_header: HeaderName::from_static("name_header"),
             client_version_header: HeaderName::from_static("version_header"),
+            buffer_size: 10000,
```

Comment:
NIT:
```suggestion
            buffer_size: default_buffer_size(),
```
Note: You'll have to make `default_buffer_size()` accessible here...